specification/cognitiveservices/data-plane/CustomImageSearch/stable/v1.0/CustomImageSearch.json

```diff
@@ -104,9 +104,7 @@
             "in": "query",
             "description": "The identifier for the custom search configuration",
             "required": true,
-            "type": "integer",
-            "format": "int64",
-            "minimum": 0
+            "type": "string"
```

Comment:
Why is this being changed in a stable swagger spec. This will break existing SDKs

Reply:
We are changing it from the api side. This parameter is no longer an uint, it's a guid, but it is backward compatible so uint will still work. Docukentation for the API has also been updated